vizro-ai/examples/dashboard_ui/actions.py

```diff
@@ -75,20 +106,25 @@ def create_response(ai_response, figure, user_prompt, filename):
             api_base=api_base,
             vendor_input=vendor_input,
         )
-        ai_code = ai_outputs.code
-        figure = ai_outputs.get_fig_object(data_frame=df)
+        ai_code = ai_outputs.code_vizro
+        figure_vizro = ai_outputs.get_fig_object(data_frame=df, vizro=True)
+        figure_plotly = ai_outputs.get_fig_object(data_frame=df, vizro=False)
         formatted_code = black.format_str(ai_code, mode=black.Mode(line_length=100))
+        ai_code_outputs = {
+            "vizro": {"code": ai_outputs.code_vizro, "fig": figure_vizro.to_json()},
```

Comment:
Does `figure_vizro.to_json()` be used at all?

Reply:
Yes, its used when downloading html static file of the produced fig. `figure_plotly.to_json()` is used to download fig as json.